### PR TITLE
Feat(cb2-6547): create a psv desk based Test Group 4

### PR DESCRIPTION
**Pull request by naathanbrown**
## Create a PSV desk based Test Group 4

[https://dvsa.atlassian.net/browse/CB2-6547](6547)

## Checklist

- [ ] Branch is rebased against the latest develop/common
- [ ] Necessary `id` required prepended with `"test-"` have been checked with automation testers and added
- [ ] Code and UI has been tested manually after the additional changes
- [ ] PR title includes the JIRA ticket number
- [ ] Squashed commits contain the JIRA ticket number
- [ ] Link to the PR added to the repo
- [ ] Delete branch after merge

### CODE REVIEW

File: src/app/forms/templates/test-records/section-templates/test/desk-based-test-section-group4.template.ts [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+              value: null,
+              required: true
+            },
+            {
+              name: 'certificateNumber',
+              label: 'Certificate number',
+              type: FormNodeTypes.CONTROL,
+              width: FormNodeWidth.L,
+              validators: [{ name: ValidatorNames.Alphanumeric }],
+              value: null
```
Comment (Ruairidh-BJSS) [inline src/app/forms/templates/test-records/section-templates/test/desk-based-test-section-group4.template.ts:68]: The confluence page says this is automatically created... can this be hidden?

Comment (naathanbrown) [reply src/app/forms/templates/test-records/section-templates/test/desk-based-test-section-group4.template.ts:68]: Thanks, done this

File: src/app/forms/templates/test-records/create-master.template.ts [hunk 2]
```diff
@@ -145,6 +146,17 @@ export const contingencyTestTemplates: Record<VehicleTypes, Record<string, Recor
       defects: defectsHiddenSection,
       customDefects: CustomDefectsHiddenSection,
       reasonForCreation: reasonForCreationSection
+    },
+    testTypesDeskBasedGroup4: {
+      required: CreateRequiredSection,
+      vehicle: DeskBasedVehicleSectionDefaultPsvHgv,
```
Comment (tahayazici) [inline src/app/forms/templates/test-records/create-master.template.ts:152]: `DeskBasedVehicleSectionDefaultPsvHgv` only contains VIN, VRM, Country of registration & EU Vehicle Category. Whereas you also need Odometer reading & Odometer units.

I'd make a new template for this; I couldn't find one that conforms to your requirements.

Comment (mcadenas-bjss) [reply src/app/forms/templates/test-records/create-master.template.ts:152]: You might have old code Taha.
image

**Review by mcadenas-bjss — APPROVED** [review]

**Review by HazelWilderspin — APPROVED** [review]

**Review by mcadenas-bjss — APPROVED** [review]

**Pull request merged**